cli/schemas/config-file.v1.json

```diff
@@ -198,9 +198,31 @@
       }
     },
     "importMap": {
-      "description": "The location of an import map to be used when resolving modules. If an import map is explicitly specified, it will override this value.",
+      "description": "The location of an import map to be used when resolving modules. If an import map is specified as a flag or using \"imports\" and \"scopes\" properties, they will override this value.",
```

Comment:
This description is the wrong way around according to your comment about precedence: This should override `imports` and `scopes` but not a specified flag.

Reply:
Good catch